desc/objectives/_coils.py

```diff
@@ -1267,6 +1283,197 @@ def compute(self, field_params, constants=None):
         return f
 
 
+class QuadraticFluxMinimizingSurface(_Objective):
+    """Target B*n = 0 on a surface.
+
+    Used to find a quadratic-flux-minimizing (QFM) surface, so a
+    `FourierRZToroidalSurface` should be passed to the objective.
+    Should always be used along with a ``ToroidalFlux`` objective to
+    ensure that the resulting QFM surface has the desired amount of
+    flux enclosed and avoid trivial solutions.
+
+    Parameters
+    ----------
+    surface :  FourierRZToroidalSurface
+        QFM surface upon which the normal field error will be minimized.
+    field : MagneticField
+        External field produced by coils or other source, which will be optimized to
```

Comment:
Maybe mention `field_fixed` argument here to clarify that this doesn't always change? And also for `surface`, mention it is always allowed to change